src/screens/FreezeFunds/04-ValidationSuccess.js

```diff
@@ -51,6 +34,7 @@ type Props = {
 
 const ValidationSuccess = ({ account, navigation }: Props) => {
   const time = useTimer(60);
```

Comment:
if we're not using this we should remove it :+1: since you remove the useTimer as well

Reply:
we are using it 🤔